config/config.go

```diff
@@ -358,7 +364,9 @@ type APIServer struct {
 	API         APIConfigParameters `validate:"required"`
 	PostgreSQL  PostgreSQL          `validate:"required"`
 	Coordinator struct {
-		API struct {
+		// ForgerAddress is the address under which this coordinator is forging
+		ForgerAddress ethCommon.Address `validate:"required"`
```

Comment:
This parameter should have an env variable option to let the user confure it  as environmental variable